AutoFlow_OmicsDataHandling/INCA_script_generator.py

```diff
@@ -0,0 +1,1232 @@
+                )
+                tracer_met_atompositions = prepare_input(
+                    tracer["met_atompositions"], "Curly"
+                )
+                tracer_met_elements = prepare_input(
+                    tracer["met_elements"], "Curly"
+                )
+                for cnt, met_atompositions in enumerate(
+                    tracer_met_atompositions
+                ):
+                    tmp_script = (
+                        tmp_script
+                        + tracer_met_elements[cnt]
+                        + str(met_atompositions)
+                        + " "
+                    )
+                # remove extra whitespace
+                tmp_script = tmp_script[:-1]
+                tmp_script = tmp_script + "';...\n"
+        tmp_script = tmp_script + "});\n"
+        tmp_script = (
+            tmp_script + "\n% define fractions of tracers used\nt.frac = [ "
+        )
+        for tracer_cnt, tracer in tracer_I.iterrows():
+            if tracer["experiment_id"] == experiment:
+                tmp_script = tmp_script + str(tracer["ratio"]) + ","
+        # remove extra comma
+        tmp_script = tmp_script[:-1]
+        tmp_script = tmp_script + " ];\n"
+        mat_script = mat_script + tmp_script
+
+        # write flux measurements
+        tmp_script = ""
+        tmp_script = (
+            tmp_script + "\n% define experiments for fit data\nf = data(' "
+        )
+        for flux_cnt, flux in measuredFluxes_data_I.iterrows():
+            if flux["experiment_id"] == experiment:
+                tmp_script = tmp_script + flux["rxn_id"] + " "
+        tmp_script = tmp_script[:-1]
+        tmp_script = tmp_script + " ');\n"
+        tmp_script = tmp_script + "\n% add fit values\nf.val = [...\n"
+        for flux_cnt, flux in measuredFluxes_data_I.iterrows():
+            if flux["experiment_id"] == experiment:
+                tmp_script = tmp_script + str(flux["flux_average"]) + ",...\n"
+        tmp_script = tmp_script + "];\n"
+        tmp_script = tmp_script + "% add fit stds\nf.std = [...\n"
+        for flux_cnt, flux in measuredFluxes_data_I.iterrows():
+            if flux["experiment_id"] == experiment:
+                tmp_script = tmp_script + str(flux["flux_stdev"]) + ",...\n"
+        tmp_script = tmp_script + "];\n"
+        tmp_script = (
+            tmp_script
+            + "\n% initialize experiment with t and add f and d\nx = experiment(t);\n" # noqa E501
+        )
+        tmp_script = tmp_script + "x.data_flx = f;\n"
+        tmp_script = tmp_script + "x.data_ms = d;\n"
+        tmp_script = (
+            tmp_script
+            + "\n% assing all the previous values to a specific experiment"
+        )
+        tmp_script = tmp_script + "\nm.expts(%d) = x;\n" % (experiment_cnt + 1)
+
+        mat_script = mat_script + tmp_script
+    return mat_script, fragments_used
+
+
+def mapping(experimentalMS_data_I, fragments_used):
+    # check stedv stuff
```

Comment:
I think this comment will break the docstrings rendering? I would suggest to remove or add to the docstrings itself.